packages/react-jsx-highcharts-gantt/src/components/YAxis/YAxis.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { YAxis } from 'react-jsx-highcharts';
+
+const GanttYAxis = ({ categories, type, ...props }) => {
+  const seriesType = categories ? type : 'treegrid';
+  return (
+    <YAxis {...props} id="yAxis" dynamicAxis={false} type={seriesType} />
+  );
+}
+
+GanttYAxis.defaultProps = {
+  grid: { enabled: true },
+  reversed: true,
+  staticScale: 50,
+  visible: false
```

Comment:
Apologies I made a mistake here, can you remove the visible default prop from both axes. 

They should both be [visible true](https://api.highcharts.com/gantt/yAxis.visible) (which is already default)